Scripts/datatypes/purpose.py

```diff
@@ -164,6 +164,15 @@ def transform_impedance(self, impedance):
                                                     cost.car_drv_occupancy[self.name])
                     except KeyError:
                         pass
+        for name in cost.gen_cost:
+            if name == self.name:
+                for mode in cost.gen_cost[name]:
+                    try:
+                        vot = cost.gen_cost[name][mode]
+                        day_imp[mode]["gen_cost"] = (day_imp[mode]["time"] / 60 * vot 
+                                                     + day_imp[mode]["cost"])
+                    except KeyError:
+                        pass
```

Comment:
I think we should implement time and cost parameters as regular json model parameters instead. For the model user it is not relevant whether the model is estimated with fixed value of time or not. Hence, this extra script does not add any value in my opinion.

Reply:
Could be, but I think we can sort this out later if we want to merge. This is more convenient way when testing demand model specification.